cgi-bin/DW/Controller/Admin/SupportCat.pm

```diff
@@ -113,7 +113,7 @@ sub category_controller {
             if $cat->{basepoints} < 0 || $cat->{basepoints} > 255;
         if ( $cat->{replyaddress} ) {
             my @errors = ();
-            LJ::check_email( $cat->{replyaddress}, \@errors );
+            LJ::check_email( $cat->{replyaddress}, \@errors, $post_args, \( $vars->{email_checkbox} ) );
```

Comment:
Maybe I haven't been doing enough Perl but what does the `\( ... )` syntax do?

Reply:
Oh this just takes a reference to the scalar inside? I see!

Reply:
Yep, that :-)